src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestDfpParamsPresentMobile.java

```diff
@@ -13,26 +14,43 @@
  * @link https://www.google.com/dfp/5441#delivery/LineItemDetail/LINE_ITEM_ID=115974612
  * @ownership AdEngineering
  */
-public class TestDfpParamsPresentMobile extends TemplateNoFirstLoad {
+public class TestDfpParamsPresentMobile extends MobileTestTemplate {
 
   private static final String LINE_ITEM_ID = "115974612";
   private static final String CREATIVE_ID = "50006703732";
 
   @Test(
       dataProviderClass = MobileAdsDataProvider.class,
-      dataProvider = "dfpParams",
-      groups = {"MobileAds", "TestDfpParamsPresentMobile_GeoEdgeFree"}
+      dataProvider = "dfpParamsSynthetic",
+      groups = {"MobileAds", "TestDfpParamsPresentSyntheticMercury"}
   )
-  public void testDfpParamsPresentMobile_GeoEdgeFree(String wikiName,
-                                                     String article,
-                                                     String adUnit,
-                                                     String slot,
-                                                     List<String> pageParams,
-                                                     List<String> slotParams) {
+  public void testDfpParamsPresentSyntheticMercury(String wikiName,
+                                                   String article,
+                                                   String adUnit,
+                                                   String slot,
+                                                   List<String> pageParams,
+                                                   List<String> slotParams) {
     String testedPage = urlBuilder.getUrlForPath(wikiName, article);
     AdsBaseObject ads = new AdsBaseObject(driver, testedPage);
     ads.verifyGptIframe(adUnit, slot, "mobile");
     ads.verifyGptParams(slot, pageParams, slotParams);
     ads.verifyGptAdInSlot(slot, LINE_ITEM_ID, CREATIVE_ID);
   }
+
+  @Test(
+      dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "dfpParams",
+      groups = {"MobileAds", "TestDfpParamsPresentMercury"}
+  )
+  public void testDfpParamsPresentMercury(String wikiName,
+                                          String article,
+                                          String adUnit,
+                                          String slot,
+                                          List<String> pageParams,
+                                          List<String> slotParams) {
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    AdsBaseObject ads = new AdsBaseObject(driver, testedPage);
+    ads.verifyGptIframe(adUnit, slot, "mobile");
+    ads.verifyGptParams(slot, pageParams, slotParams);
```

Comment:
@drets: I wouldn't  recommend that - this way it's more flexible and informative, it's a good practice to get rid of "goDoMagic", "makeSureEverythingIsWorkingAsExpected" methods. In this case more verbosity is fine.